docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,199 @@
+
+If the pull request fixes a bug:
+
+* The pull request description must include `Fixes #<issue number>`.
+* To avoid regressions, the pull request should include tests that replicate the fixed bug.
+
+The owning team _squashes_ all commits into one when we accept a pull request. The title of the pull request becomes the subject line of the squashed commit message. We still encourage contributors to write informative commit messages, as they become a part of the Git commit body.
+
+We use the pull request title when we generate change logs for releases. As such, we strive to make the title as informative as possible.
+
+Make sure that the title for your pull request uses the same format as the subject line in the commit message.
+
+### Quality Gates for pull requests
+
+GitHub Actions are used to enforce quality gates when a pull request is created or when any commit is made to the pull request. These GitHub Actions enforce our minimum code quality requirement for any code that gets checked into the code repository. If any of the quality gates fail, it is expected that the contributor will look into the check log, understand the problem and resolve the issue. If help is needed, please feel free to reach out to the maintainers of the project for [support](SUPPORT.md).
+
+#### Code build/test/coverage
+
+[GitHub action](https://github.com/ansible-collections/dellemc.enterprise_sonic/actions) that runs unit tests automatically and checks that the code coverage of each package meets a configured threshold (currently 90%). An error is flagged if a given pull request does not meet the test coverage threshold and blocks the pull request from being merged. When it fails, it is expected that the contributor will look into the log, understand the problem and resolve the issue.  
+
+## Code Reviews
+
+All submissions, including submissions by project members, require review. We use GitHub pull requests for this purpose. Consult [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more information on using pull requests.
+
+A pull request must satisfy the following for it to be merged:
+
+* A pull request will require at least 2 maintainer approvals.
+* Maintainers must perform a review to ensure the changes adhere to guidelines laid out in this document.
+* If any commits are made after the PR has been approved, the PR approval will automatically be removed and the above process must happen again.
+
+## Code Style
+
+For the Python code in the repository, we expect the code styling outlined in [Ansible python guide]( https://docs.ansible.com/ansible/latest/dev_guide/developing_python_3.html). In addition to this, we have the following supplements:
+
+* Contributions should adhere to ansible Coding standard guidelines as we follow these standards.
+* Should include [test](https://github.com/ansible-collections/dellemc.enterprise_sonic/tree/collections/tests) scripts for your changes.
```

Comment:
This link is incorrect. Instead, please use the following link if it points to the intended location:

https://github.com/ansible-collections/dellemc.enterprise_sonic/tree/main/tests